Geometry/VeryForwardData/data/RP_Materials/v5/RP_Materials.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
```

Comment:
The only change for this file is the deletion of unused composite material `PPS_Epoxy`.